### PR TITLE
More data research based on the city open data

### PR DESCRIPTION
**Pull request by zzptichka**
- wading pools
- splash pads
- basketball courts
- green space

### CODE REVIEW

File: Green-space/clean.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+"use strict";
+const turf = require("@turf/turf");
+const multipolygon = require("turf-multipolygon");
```
Comment (DenisCarriere) [inline Green-space/clean.js]: Inside `@turf/turf` you'll have `multiPolygon` as a function.

**Comment by DenisCarriere** [comment]
Cool stuff! 👍 I'll add you as Admin to this repo to make commits the master branch

File: Green-space/clean.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const fs = require("fs");
+const geojson2osm = require('geojson2osm');
+const reader = require('geojson-writer').reader
+
+const collection1 = turf.featureCollection([]);
+
+let pools = reader('parks-source.json')
+
+pools.features.map(result => {
```
Comment (DenisCarriere) [inline Green-space/clean.js]: Using `forEach()` would be more appropriate for this operation since `map()` it continuously builds an Array in memory.

Comment (zzptichka) [reply Green-space/clean.js]: Didn't I borrow this from your code? :)

File: Green-space/clean.js (hunk 1 above)
Comment (zzptichka) [reply Green-space/clean.js]: I think I was getting "not a function" error. Must be something stupid like case-sensitivity...

File: Green-space/clean.js (hunk 2 above)
Comment (DenisCarriere) [reply Green-space/clean.js]: Probably! You shouldn't use bad references 🤣

File: Green-space/clean.js (hunk 1 above)
Comment (DenisCarriere) [reply Green-space/clean.js]: Oh it is case sensitive, which Text editor are you using? TurfJS works really well with VSCode which allows for IntelliSense (autocomplete helper).

**IntelliSense in Visual Studio Code**

image

Comment (zzptichka) [reply Green-space/clean.js]: I'm using atom. There is probably intellisense-like plugin. Will check it out.

**Pull request merged**